### DISCUSSION TITLE
Environment dependent seed files

### DISCUSSION THREAD

**Discussion by blchelle** [prisma/prisma#12455] [Q&A] [answered]
Is there any way to have multiple seed files that run dependent on the environment. I have two sets of data, one for seeding a development and one for seeding a production environment.

I'm imagining something like below in my package.json 
```json
  "prisma": {
    "seed:dev": "ts-node prisma/seeds/dev.ts",
    "seed:prod": "ts-node prisma/seeds/prod.ts"
  },
```
But I don't believe there's a way prisma to know which of these to run after a `prisma migrate dev` or a `prisma reset dev`. Any ideas for how to do this are more than welcome. Thank you! 

Some additional notes are that I have .env files for both development and production and migrate scripts that look like
```json
  "scripts": {
    "prisma:migrate:dev": "dotenv -e .env.development -- prisma migrate dev",
    "prisma:migrate:prod": "dotenv -e .env.production -- prisma migrate dev",
    "prisma:reset:dev": "dotenv -e .env.development -- prisma migrate reset dev",
    "prisma:reset:prod": "dotenv -e .env.production -- prisma migrate reset dev"
  },
```

**Comment by austincrim** [comment, marked as answer]
Hey @blchelle 👋,

This looks like a great idea for a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=). Feel free to convert this discussion to an issue or create one from scratch. Our engineers will pick it up from there.

I'm curious, what is your use case for seeding a production environment?

Thanks!

**Reply by blchelle** [reply]
My use case is populating a table of text passages for a multiplayer typing game similar to [Typeracer](https://typeracer.com). The data is mostly static and could be stored elsewhere but including it in a RDBMS was the simpler option in my case.

**Comment by RamonSerraVasconcelos** [comment]
Any updates?

**Comment by Horlando-Leao** [comment]
Any updates?

**Reply by ced-mos** [reply]
@blchelle Have you converted this request to a feature request? I haven't found it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
